### PR TITLE
testing/gnome-shell: upgrade to 3.32.1

### PR DESCRIPTION
**Pull request by Cogitri**
~~Depends on #7241~~
~~Depends on #7258~~
~~Depends on #7261~~
~~Depends on #7262~~
~~Depends on #7374~~
~~Depends on #7375~~
~~Depends on #7376~~
~~Depends on #7377~~
~~Depends on #7417~~

### CODE REVIEW

**Comment by Cogitri** [comment]
FWIW, this needs my other PRs to be merged first. Please also note that while this does build just fine for me, gdm gets stuck during startup and gnome-shell crashes during startup.

**Comment by tcely** [comment]
Adding `Depends on` lines would help me to know which other PRs I should be pulling to build this.

**Comment by Cogitri** [comment]
All of these https://github.com/alpinelinux/aports/pulls/Cogitri but virt-manager and cargo-bootstrap

**Comment by Cogitri** [comment]
So #7241 #7258 #7261 #7262 #7374 #7375 #7376 #7377 #7379 #7417

**Comment by Ikke** [comment]
#7262 Is the only one left that's not merged yet.

**Comment by Cogitri** [comment]
Yup, thanks for the reviewing :)

**Comment by Cogitri** [comment]
This passes CI now! :)

File: community/network-manager-applet/APKBUILD [hunk 1]
```diff
@@ -15,12 +16,11 @@ makedepends="
...
-	libsecret-dev
-	libsecret-dev
 	mobile-broadband-provider-info
 	modemmanager-dev
 	networkmanager-dev
 	polkit-dev
+	gobject-introspection-dev
 	"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
 source="https://download.gnome.org/sources/network-manager-applet/${pkgver%.*}/network-manager-applet-$pkgver.tar.xz
```
Comment (TBK) [inline community/network-manager-applet/APKBUILD]:
```suggestion
source="https://download.gnome.org/sources/network-manager-applet/${pkgver%.*}/network-manager-applet-$pkgver.tar.xz"
```

Comment (TBK) [reply community/network-manager-applet/APKBUILD]: Any tests?

File: testing/gnome-session/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+arch="all !aarch64 !armhf !armv7 !s390x"
+license="GPL-2.0-or-later"
+depends="gnome-shell polkit alsa-plugins-pulse dconf pulseaudio-alsa"
+makedepends="gnome-settings-daemon-dev libsm-dev
+	elogind-dev itstool libxslt libxml2-utils json-glib-dev gnome-desktop-dev
+	gtk+3.0-dev startup-notification-dev upower-dev meson py3-setuptools xmlto
+	docbook-xml"
+subpackages="$pkgname-lang $pkgname-doc"
+source="https://download.gnome.org/sources/$pkgname/${pkgver%.*}/$pkgname-$pkgver.tar.xz"
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (TBK) [inline testing/gnome-session/APKBUILD]:
```suggestion

```

Comment (TBK) [reply testing/gnome-session/APKBUILD]: Any tests?

File: community/network-manager-applet/APKBUILD (hunk 1 above)
Comment (Cogitri) [reply community/network-manager-applet/APKBUILD]: Thanks, fixed!

**Comment by Cogitri** [comment]
Pushed fixed version, thanks for the review :)

**Comment by TBK** [comment]
3 last things:
```
>>> WARNING: gdm-dev*: Found static archive on usr/lib/libgdm.a but name doesn't end with -static
>>> gdm-lang*: Running split function lang...
>>> gdm-lang*: Preparing subpackage gdm-lang...
>>> gdm-lang*: Running postcheck for gdm-lang
>>> gdm*: Running postcheck for gdm
>>> WARNING: gdm*: Found OpenRC directory (/etc/conf.d or /etc/init.d) but name doesn't end with -openrc 
```

mutter does not seem to have a test suite:
```
>>> mutter: Entering fakeroot...
ninja: Entering directory `output'
[0/1] Installing files. 
```

**Comment by Cogitri** [comment]
Fixed, thanks.

**Comment by TBK** [comment]
I had to disable mutter on ppc64le because of - https://build.alpinelinux.org/buildlogs/build-edge-ppc64le/testing/mutter/mutter-3.32.2-r0.log
```
../cogl/cogl-path/cogl-path.h:55:10: fatal error: cogl-path/cogl-path-enum-types.h: No such file or directory
 #include <cogl-path/cogl-path-enum-types.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
As a result I also had to disable gnome-shell and gdm on ppc64le

**Pull request closed without merging**